library/src/lib/scroll-spy/scroll-spy.directive.ts

```diff
@@ -1,26 +1,46 @@
 import { Directive, ElementRef, EventEmitter, HostListener, Input, Output } from '@angular/core';
 
+/**
+ * A directive designed to help navigation elements determine the element currently in view of the user. 
+ */
 @Directive({
     selector: '[fdScrollSpy]'
 })
 export class ScrollSpyDirective {
-
+    /**
+     * An array of tags to track.
+     */
     @Input()
     public trackedTags: string[] = [];
 
+    /**
+     * Whether an arrow is displayed when the user scrolls to the absolute top or bottom.
+     * To hide the arrow, set the property to *true*.
+     */
     @Input()
     public fireEmpty: boolean = false;
```

Comment:
There is no visual aspect to this component. All this input does is determine whether events are still fired if there is no tag present on the user's screen. It *can* be used to do as you describe, but that's up to the user.

Reply:
Done